app/static/js/app.js

```diff
@@ -346,6 +347,12 @@ document
 document.getElementById("cancel-shutdown").addEventListener("click", () => {
   hideElementById("shutdown-confirmation-panel");
 });
+document.getElementById("shutdown-confirmation-panel").addEventListener("click", (e) => {
+  e = window.event || e;
+  if(e.target.className === "overlay") {
```

Comment:
The spacing here is causing the build check to fail. Could you run the formatter so it matches the JS/CSS whitespace conventions?

```bash
npm install npm install prettier@2.0.5
./node_modules/.bin/prettier --write app/static
```